com.trollworks.gcs/src/com/trollworks/gcs/preferences/ReferenceLookupPreferences.java

```diff
@@ -94,8 +158,25 @@ public boolean isSetToDefaults() {
 
     @Override
     public void reset() {
+        Preferences.getInstance().clearPdfLaunchString();
+        mPdfViewerCommandLine.setText("");
         Preferences.getInstance().clearPdfRefs();
         mPanel.removeAll();
         mPanel.setSize(mPanel.getPreferredSize());
     }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
```

Comment:
I don't believe you needed to add ActionListener to the interfaces this class implements. Can you point out where this is required? If not, please remove it and the implements ActionListener clause, above.